src/NuGet.Clients/NuGet.VisualStudio.Client/.vsixignore

```diff
@@ -14,7 +14,6 @@ Microsoft.Internal.VisualStudio.Interop.dll
 Microsoft.Internal.VisualStudio.Shell.Framework.dll
 Microsoft.IO.Redist.dll
 Microsoft.NET.StringTools.dll
-Microsoft.ServiceHub.Client.dll
```

Comment:
There should be binding redirects for this. How certain are you that we need it?

Reply:
The custom code we have in the csproj that reads the `vsixignore` file was failing the build unless I removed the line. Note that the PR doesn't add it into `vsixinclude`, so we're not including it in the vsix.

I think this scenario (vsixignore lists a file that isn't in the bin directory) used to be a warning, but from memory Jeff did something to treat all msbuild warnings as errors. Maybe that's why it caused the build failure? In any case, this PR won't build unless the line is deleted.

Reply:
Great point, resolving.